understanding/22/accessible-authentication-enhanced.html

```diff
@@ -12,6 +12,16 @@ <h1>Understanding Accessible Authentication (Enhanced)</h1>
         <h2>Status</h2>
         <p>This understanding document is part of the <a href="https://w3c.github.io/wcag/guidelines/22/"><strong>draft</strong> WCAG 2.2 content</a>. It may change or be removed before the final WCAG 2.2 is published.</p>
     </section> 
+   
+    <section id="brief">
+        <h2>In brief</h2>
+        <dl>
+            <dt>Goal</dt><dd>Make logins possible with less mental effort.</dd>
+            <dt>What to do</dt><dd>Don’t make people recognize objects or recall personal information to login.</dd>
```

Comment:
"personal information" makes me think of the name/email/phone-number that we explicitly allow recall tests for - the understanding document allows them specifically *because* they are "personal to the user". Maybe "memorize information" instead of "recall personal information"?

Reply:
I  want to set the context that for almost all these SCs, the language of the first two lines is not being changed. It's being repositioned, and a third line added.
So, while I think this round gives us a chance to revisit existing wording -- _especially_ in a case like this where the wording _has_ been changed for this PR -- I want to emphasize that the third line (and overall repositioning) is the focus of the PR (and I get that the repositioning may motivate some of these comments).

Just trying to manage expectations :)

Reply:
We previously had "user-supplied information" approved when we adopted the In Brief, and we can just fall back to that -- although this modification was based on user feedback AND arguably the prior text has the same potential for confusion

The wording of the SC is 
> Personal Content
The cognitive function test is to identify non-text content the user provided to the website.

with a note that non-text content "may be represented by images, video, or audio."

How about 
```suggestion
            <dt>What to do</dt><dd>Don’t make people recognize objects or user-provided images and videos to login.</dd>
```

Reply:
"personal content" isn't clear to me.

Reply:
Okay, falling back to previously approved language, let's make it
```suggestion
            <dt>What to do</dt><dd>Don’t make people recognize objects or user-supplied images to login.</dd>
```